libbuteosyncfw/profile/SyncSchedule.cpp

```diff
@@ -166,12 +166,12 @@ QString SyncSchedule::toString() const
     return doc.toString(PROFILE_INDENT);
 }
 
-DaySet SyncSchedule::days() const
+SyncSchedule::Days SyncSchedule::days() const
 {
     return d_ptr->iDays;
 }
 
-void SyncSchedule::setDays(const DaySet &aDays)
+void SyncSchedule::setDays(const SyncSchedule::Days &aDays)
```

Comment:
This and the rush version could be also by-value now.

Reply:
Done here and for rush days also.